overview.html

```diff
@@ -0,0 +1,452 @@
+<html>
+<!--
+This overview.html provides Javadoc overview for the application.
+The content from the body section is added to the Javadoc overview page.
+There is no need to add a head/title as that is provided by Javadoc.
+-->
+<body>
+
+<P>
+    This is the official Cloudant library for java.
```

Comment:
Oh and also something about how it can be used with couch, do we have anything which discusses that already - plain couch vs Cloudant features etc?